src/components/MainPageComponent.tsx

```diff
@@ -11,10 +27,105 @@ function MainPageComponent() {
         border: '4px solid',
         borderColor: theme.palette.primary.main,
         borderRadius: '4px',
-        height: '500px',
+        padding: '1rem',
       }}
     >
-      <Typography>Hello World</Typography>
+      {/* Grid container header of charlist, contains avatar and character name */}
+      <Grid
+        container
+        flexDirection="row"
+        justifyContent="space-between"
+        mb="1rem"
+      >
+        <Grid item xs={2}>
+          <Avatar sx={boxSx}></Avatar>
+        </Grid>
+        <Grid item xs={10}>
+          <Typography sx={{ fontSize: '5rem' }}>Hello World</Typography>
+        </Grid>
+      </Grid>
+      {/* Grid container with character stats */}
+      <Grid
+        container
+        flexDirection="row"
+        justifyContent="space-between"
+        mb="1rem"
+      >
+        {Array.from(Array(7)).map((_, index) => (
+          <Grid item xs={1.5} key={index}>
+            <Box sx={boxSx}></Box>
+          </Grid>
+        ))}
+      </Grid>
+      <Grid container>
+        {/* Grid container with character skills */}
+        <Grid
+          container
+          flexDirection="row"
+          columnSpacing={3}
+          justifyContent="space-between"
+          mb="1rem"
+        >
+          <Grid item xs={4}>
+            <Grid container height="100%" flexDirection="column">
+              {Array.from(Array(17)).map((_, index) => (
+                <Grid item key={index}>
+                  <Grid
+                    container
+                    flexDirection="row"
+                    justifyContent="space-between"
+                  >
+                    {Array.from(Array(3)).map((_, index) => (
+                      <Box sx={{ ...boxSx }} key={index}></Box>
```

Comment:
Remove useless copying of styles object, as already done on line 56 (same goes for every other `sx={{...boxSx}}` on page)

Reply:
My bad, was covered on [this comment](https://github.com/MangriMen/Paramoios/pull/35/files/00f024d12990bca29519566ea42781b00f7f4109#r874176648)